include/grpc/impl/channel_arg_names.h

```diff
@@ -328,6 +328,9 @@
 /** Channel arg to set http proxy per channel. If set, the channel arg
  *  value will be preferred over the environment variable settings. */
 #define GRPC_ARG_HTTP_PROXY "grpc.http_proxy"
+/** Channel arg to set proxy per channel. If set, the channel arg
+ *  value will be preferred over the environment variable settings. */
+#define GRPC_ARG_ADDRESS_PROXY "grpc.address_proxy"
```

Comment:
I think the arg name needs to be more descriptive of what it does, and the comment needs to be clearer as well.  I suggest something like this:

```
/** Specifies an HTTP proxy to use for individual addresses.
 *  The proxy must be specified as an IP address, not a DNS name.
 *  If set, the channel arg value will be preferred over the environment
 *  variable settings. */
#define GRPC_ARG_ADDRESS_HTTP_PROXY "grpc.address_http_proxy"
```

Reply:
Done